tests/test_api.py

```diff
@@ -86,3 +88,300 @@ def test_protected_route(self):
         self.assertEqual(response2.status_code, 401)
         response3 = self.app.get('/api/user', headers=bad_headers)
         self.assertEqual(response3.status_code, 401)
+
+
+class TestAPIAlchemy(TestCase):
+
+    def create_app(self):
+        app.config['TESTING'] = True
+        return app
+
+    def setUp(self):
+        db.create_all()
+
+    def tearDown(self):
+        db.session.remove()
+        db.drop_all()
+
+    def test_check_game(self):
+        user = User(email='test@test.com', name='test', password='tester')
```

Comment:
We probably need the email to generate randomly

Reply:
Or are these just getting a user that's already created? Do we have a function which makes sure they are created?

Reply:
There needs to be a user and game already created to be able to create a player object. It doesn't really matter what they are beyond that.

Reply:
We will need to create the user/game objects because when this runs in memory in CI they won't be any previous data